### PR TITLE
feat: added event_id to alertDTO to utilize as unique identifier in workflows and rest

### PR DESCRIPTION
**Pull request by pehlicd**
## 📑 Description
In current stack, there is nothing evaluate alerts with unique id, so I've updated `event_id` to utilize in workflows.

## ✅ Checks

- [x] My pull request adheres to the code style of this project
- [ ] My code requires changes to the documentation
- [ ] I have updated the documentation as required
- [ ] All the tests have passed

### CODE REVIEW

File: keep/api/core/db.py [hunk 1]
```diff
@@ -996,6 +996,7 @@ def get_last_alerts(tenant_id, provider_id=None, limit=1000) -> list[Alert]:
         alerts = []
         for alert, startedAt in alerts_with_start:
             alert.event["startedAt"] = str(startedAt)
+            alert.event["event_id"] = str(alert.id)
```
Comment (talboren) [inline keep/api/core/db.py:999]: I'd suggest the following so we don't override something that may be provided by the provider itself
```suggestion
            alert.event["keep_event_id"] = str(alert.id)
```

Comment (pehlicd) [reply keep/api/core/db.py]: done!

File: keep/api/routes/alerts.py [hunk 2]
```diff
@@ -523,8 +523,10 @@ def handle_formatted_events(
                 alert_hash=formatted_event.alert_hash,
             )
             session.add(alert)
-            formatted_event.event_id = alert.id
-            alert_dto = AlertDto(**alert.event)
+            session.flush()
+            session.refresh(alert)
```
Comment (talboren) [inline keep/api/routes/alerts.py:527]: why not before?

**Review by talboren — APPROVED** [review]
LGTM

**Pull request merged**